pupyl/cli.py

```diff
@@ -0,0 +1,28 @@
+""" Manage cli arguments"""
+
+import argparse
+
+
+class PupylCommandLineInterface:
+    def __init__(self):
+        self.parser = argparse.ArgumentParser(
+            description='Manage pupyl cli arguments.'
+        )
+        self.subparsers = self.parser.add_subparsers(dest='command_name')
```

Comment:
@92amartins, running the test suites, seems like there's an issue on subparsers:

```python
usage: py.test [-h] {index,serve} ...
py.test: error: cannot have multiple subparser arguments
```

Reply:
Thanks! I'll have a look.